src/datasources/data-frame/query_help.md

```diff
@@ -0,0 +1,74 @@
+# SystemLink Data Frames data source
+
+## 📊 Overview
+
+The **SystemLink Data Frames** data source allows you to display data table properties and visualize data from multiple data tables in dashboards.
+
+---
+
+## 🎯 Key Features
+
+* **Flexible Filtering**: Filter data tables by result, data table, and column properties.  
+* **Display Table Metadata**: Retrieve and display metadata properties such as name, ID, row count, column information, and workspace.  
+* **Display Data Table Data**: Retrieve and display row data from selected columns across one or more filtered data tables.  
+
+---
+
+## ⚠️ Limitations
+
+### Required Filters
+At least one filter (data table, test result, or column filter) must be specified. Queries without any filters will return no results.
+
+### Data Limits
+
+| Limit Type           | Maximum               | Behavior                                                                                   |
+| -------------------- | --------------------- | ------------------------------------------------------------------------------------------ |
+| **Data Tables**      | 1,000 tables          | If your filters match more than 1,000 tables, only the first 1,000 will be processed.      |
+| **Columns options**  | 10,000 unique columns | If more columns exist, only the first 10,000 are shown in the dropdown.                    |
+| **Column Selection** | 20 columns per query  | If more columns are selected, the datasource will return an error and will not query data. |
```

Comment:
Suggestion: 'If you select more than 20 columns, the data source will not query the data and instead return an error.'